primefaces/src/main/resources/META-INF/resources/primefaces/fileupload/3-fileupload.simple.js

```diff
@@ -209,9 +209,14 @@ PrimeFaces.widget.SimpleFileUpload = PrimeFaces.widget.BaseWidget.extend({
         var parameterPrefix = PrimeFaces.ajax.Request.extractParameterNamespace(this.form);
         var formData = PrimeFaces.ajax.Request.createFacesAjaxFormData(this.form, parameterPrefix, this.id, process, update);
 
+        if (this.cfg.ignoreAutoUpdate) {
+            PrimeFaces.ajax.Request.addFormData(formData, PrimeFaces.IGNORE_AUTO_UPDATE_PARAM, true, parameterPrefix);
```

Comment:
i think we should better add param to createFacesAjaxFormData

Reply:
Done